db/models.py

```diff
@@ -50,3 +57,71 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE)
```

Comment:
In case your opening parenthesis does not have arguments on the same line, your closing parenthesis should not have any arguments on the same line as well.